internal/rootcoord/quota_center.go

```diff
@@ -183,11 +192,19 @@ func (q *QuotaCenter) syncMetrics() error {
 		if err != nil {
 			return err
 		}
+		for k := range q.queryNodeMetrics {
+			delete(q.queryNodeMetrics, k)
```

Comment:
With q.clearMetrics, no need to delete here?

Reply:
for nodedown case, the nodes list decrease, so we should clear old metrics, not just insert new one